crates/story/examples/fixtures/test.lua

```
@@ -0,0 +1,130 @@
+--[[
+    HelloWorld module demonstrating Lua syntax highlighting
+    Version: 1.0.0
+--]]
+
+local VERSION = "1.0.0"
+
+-- Module configuration with default values
+local Config = {
```

Luacheck (1.2.0)

[warning] unused variable 'Config'

(W211)